editor/src/components/editor/persistence/persistence.spec.ts

```diff
@@ -34,7 +34,7 @@ let serverProjects: { [key: string]: PersistentModel } = {}
 let localProjects: { [key: string]: LocalProject<PersistentModel> } = {}
 
 const base64Contents = 'data:asset/xyz;base64,SomeBase64'
-const AssetFileWithBase64 = assetFile(base64Contents)
+const mockAssetFileWithBase64 = assetFile(base64Contents)
```

Comment:
babel-jest is more nitpicky about variable names that are in the scope of the mock.